stratum/hal/bin/barefoot/docker/build-stratum-bf-container.sh

```diff
@@ -96,36 +102,55 @@ else
   print_help
   exit 1
 fi
+if [ -t 0 ]; then
+  # Running in a TTY
+  DOCKER_OPTS+="-it "
+fi
 
-# Build Stratum BF in Docker
+# Build Stratum BF in Docker (optimized and stripped)
+EXTRA_BUILD_OPTS=""
+if [ -n "$RELEASE_BUILD" ]; then
+  # Build with optimization enabled (-O2) and with assert() calls disabled (-DNDEBUG)
+  EXTRA_BUILD_OPTS+="--compilation_mode=opt "
+  # Strip all symbols
+  EXTRA_BUILD_OPTS+="--linkopt=-Wl,--strip-all "
+fi
 set -x
-docker run --rm -it \
+docker run --rm \
   $DOCKER_OPTS \
   -v $STRATUM_ROOT:/stratum \
   -v $(pwd):/output \
   -w /stratum \
   --entrypoint bash \
   $DOCKER_IMG -c \
     "bazel build //stratum/hal/bin/barefoot:${STRATUM_TARGET}_deb \
+       $EXTRA_BUILD_OPTS \
        --define sde_ver=$SDE_VERSION \
        --define phal_with_onlp=$WITH_ONLP \
        --jobs $JOBS && \
-     cp /stratum/bazel-bin/stratum/hal/bin/barefoot/${STRATUM_TARGET}_deb.deb /output/ && \
-     cp \$(readlink -f /stratum/bazel-bin/stratum/hal/bin/barefoot/${STRATUM_TARGET}_deb.deb) /output/"
+     cp -f /stratum/bazel-bin/stratum/hal/bin/barefoot/${STRATUM_TARGET}_deb.deb /output/ && \
+     cp -f \$(readlink -f /stratum/bazel-bin/stratum/hal/bin/barefoot/${STRATUM_TARGET}_deb.deb) /output/"
 set +x
 
+
+DOCKER_OPTS=""
+if [ "$(docker version | grep Experimental | grep true | wc -l)" -eq "2" ]; then
```

Comment:
The recommended way to check for that is:

```bash
> docker version -f '{{.Server.Experimental}}'
true
```

Reply:
That is definitely cleaner.